crates/core/tedge/src/cli/connect/cli.rs

```diff
@@ -33,7 +32,7 @@ impl BuildCommand for TEdgeConnectOpt {
                 cloud: Cloud::C8y,
                 common_mosquitto_config: CommonMosquittoConfig::default(),
                 is_test_connection,
-                service_manager: service_manager(
+                service_manager: crate::system_services::service_manager(
```

Comment:
I would be good to remove the now useless `use crate::system_services::service_manager;`

Reply:
I guess the other way round would be better, not using the full path here! :+1: